src/settings.rs

```diff
@@ -33,57 +28,39 @@ impl Default for Settings {
 impl Settings {
     pub const NONE_PROFILE_ID: &'static str = "none";
 
-    /// Opens a `FileChooserDialog` to select a folder and updates
+    /// Opens a `FileDialog` to select a folder and updates
     /// the settings with the selected folder.
     pub fn select_saving_location(&self, transient_for: Option<&impl IsA<gtk::Window>>) {
```

Comment:
Since transient_for is unwrapped anyway at the function body, it could instead just take in &impl Is<gtk::Window> without the options